### PR TITLE
Create/Edit page - Unnecessary "Could not save!" message

### PR DESCRIPTION
**Pull request by rkenyon1969**
Fixed. Please use the use-case in the above thread to test.

@wski 
@akonsowski 
@clarkjefcoat

### CODE REVIEW

**Comment by rkenyon1969** [comment]
Test: 
1. Edit an existing approved listing
2. Scroll down to the icons section and delete one of the icons by pressing the trashcan
3. Don't replace the icon you just delete
4. Press the Save button
5. You get a "Could not save!" pop-up (expected)
6. Replace the deleted icon with a new icon
7. Press the Save button
8. You get a "Could not save!" pop-up (not expected)
9. The listing is updated and saved (expected)
    - “The listing changes are correctly saved, but the pop-up is confusing and wrong.”

@wski 
@akonsowski 
@clarkjefcoat

File: app/js/components/createEdit/index.jsx [hunk 1]
```diff
@@ -519,7 +519,8 @@ var CreateEditPage = React.createClass({
             scrollToError = true;
         }
         CreateEditActions.save();
-        this.setState({ scrollToError: scrollToError, lastUpdate: Date.now() });
+        //this.setState({ scrollToError: scrollToError, lastUpdate: Date.now() });
+        this.setState({ scrollToError: scrollToError }); // rjk
```
Comment (clarkjefcoat) [inline app/js/components/createEdit/index.jsx]: What does removing "lastUpdate" accomplish?  The value is used by getChildContext(), but I'm not sure how.

**Comment by clarkjefcoat** [comment]
:+1:

**Comment by wski** [comment]
👍

**Comment by tdressel** [comment]
Passed on latest

**Pull request merged**